cmd/storj-sim/network.go

```diff
@@ -177,7 +180,7 @@ func networkDestroy(flags *Flags, args []string) error {
 }
 
 // newNetwork creates a default network
-func newNetwork(flags *Flags) (*Processes, error) {
+func newNetwork(flags *Flags) (processes *Processes, err error) {
```

Comment:
Why change this?

Reply:
that was remanent of when I tried returning "cleanup" I have yet to push the update